lansa-vmss-windows-autoscale-sql-database/azuredeploy.json

```diff
@@ -0,0 +1,829 @@
+{
+   "$schema": "http://schema.management.azure.com/schemas/2015-01-01-preview/deploymentTemplate.json",
+   "contentVersion": "1.0.0.0",
+   "parameters": {
+      "msiUri": {
+         "type": "string",
+         "defaultValue": "https://lansalpcmsdn.blob.core.windows.net/app/test/AWAMAPP_v14.1.2_en-us.msi",
```

Comment:
can we include the MSI with this template instead of having it in a custom storage account? If so, please use _artifactsLocation and _artifactsLocationSasToken as described in the best practice checklist: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/bp-checklist.md